app/pom.xml

```diff
@@ -1,276 +1,291 @@
 <?xml version="1.0"?>
 <project
```

Comment:
It seems you have re-formatted this file and as a result have replaced all of the spaces with tabs.  This will break the github diff (as you can see).  Please restore the previous format and, if you can, use spaces for indentation instead of tabs for any new changes to the file.

Reply:
yep.. I thought I changed that setting to 4 spaces and didn't notice the diff. Should be ok now.